sdk/core/Azure.Core/src/Response.cs

```diff
@@ -83,8 +83,10 @@ public virtual BinaryData Content
         public abstract void Dispose();
 
         /// <summary>
-        /// Indicates whether the message's <see cref="ResponseClassifier"/> considers this
-        /// response an error.
+        /// Indicates whether the status code of the returned response is considered
+        /// an error code. This enables the caller to prevent the client from throwing
+        /// an exception when the service method takes a <see cref="RequestContext"/>
```

Comment:
We'll update this to include `or <see cref="RequestOptions"/>` if this is how we update the API to support HLCs.

Reply:
I wonder if we should not finish after the first sentence. The second sentence goes into details that are probably better described in RO docs.

Reply:
Sounds good.